src/splitBy.js

```diff
@@ -0,0 +1,24 @@
+var _curry2 = require('./internal/_curry2');
+var complement = require('./complement');
+var difference = require('./difference');
+var takeWhile = require('./takeWhile');
+
+
+/**
+ * Splits a given list at the first occurance of a satisfied predicate.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig Number -> [a] -> [[a], [a]]
+ * @param {Function} fn The predicate that determines where the array is split.
+ * @param {Array} array The array to be split.
+ * @return {Array}
+ * @example
+ *
+ *      R.splitBy(R.equals(2), [1, 2, 3]);   //=> [[1], [2, 3]]
+ */
+module.exports = _curry2(function splitBy(predicate, array) {
+  var xs = takeWhile(complement(predicate), array);
+  return [xs, difference(array, xs)];
```

Comment:
You could probably use `drop(xs.length, array)` instead of the `difference` call.

But speaking of that, I try to think of [Ramda's fundamental data type][fd] as the _list_, and the _array_ as an implementation detail.  So I would prefer to name the parameter `list`.
 

  [fd]: https://github.com/ramda/ramda/issues/96#issuecomment-47304374